Sources/PackageModel/Product.swift

```diff
@@ -41,6 +41,9 @@ public class Product {
     /// The name of the product.
     public let name: String
 
+    /// The output file name of the product.
+    public let outname: RelativePath
```

Comment:
As I commented above, we don't need to have the outname inside the target. It can be a build time thing.